src/similarity.ts

```diff
@@ -345,7 +363,9 @@ function stripExtraneousHeaders(headers: HttpHeaders): HttpHeaders {
         // Ignore.
         continue;
       default:
-        safeHeaders[key] = headers[key];
+        if (!ignoreHeaders.find((header) => header === key)) {
```

Comment:
**Suggestion**: As far as I can tell, this would be simper to implement via `contains` as we don't need the intermediary lambda?

```suggestion
        if (!ignoreHeaders.includes(key))
```